example/symbol_exec/depgraph.py

```diff
@@ -103,7 +103,7 @@
         if sat:
             for element in sol.constraints:
                 try:
-                    result = hex(sol.constraints[element].as_long())
+                    result = hex(sol.constraints[element].as_long()).strip('L')
```

Comment:
[cosmetic] Probably a better way to do this would be to use the "0x%x" % sol.constraints[element].as_long() form (probably more explicit than the "L" removing of `hex` artifact)

Reply:
Done!